gemma-core/src/main/java/ubic/gemma/persistence/util/Filter.java

```diff
@@ -194,7 +203,8 @@ public enum Operator {
         greaterThan( ">", true, null ),
         lessOrEq( "<=", true, null ),
         greaterOrEq( ">=", true, null ),
-        in( "in", true, Collection.class );
+        in( "in", true, Collection.class ),
+        inSubquery( "in", true, Subquery.class );
```

Comment:
Subqueries are not limited to `in` clauses, but that's all we need to support for now.